scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -390,21 +390,34 @@ checkPeers() {
     peerPORT=$(echo "${peer}" | cut -d: -f2)
     [[ -z ${peerIP} || -z ${peerPORT} ]] && continue
     
-    if [[ "${peerIP}" = "${lastpeerIP}" ]]; then
+    if [[ ${direction} = "out" ]]; then
+      if [[ -n ${CNCLI} && -f ${CNCLI} ]]; then
```

Comment:
Just thinking, should we instead have a flag for CNCLI usage?
So that if Flag is set to Y and CNCLI is present in $PATH? Could be helpful for comparison at times or while waiting for a fix for example

Reply:
The downside with the old method(tcptraceroute) is that from the remote cardano-node this is seen as a bad handshake connection and is logged with an error if these traces are active. So using cncli is a much cleaner way to ping as it does a proper handshake. This is why I set it as the primary method. I have done extensive testing using bad input(unresolvable dns, bad IP etc), seems to handle all of these cases gracefully.

Reply:
All good to have it at default, but would 
be good to have a flag regardless - I simply dont trust IOG to not wake up one fine day and realise they want to change the handshake as part of future HFC event

Reply:
true, adding option to disable cncli ping by flag